runtime/oti/vm_api.h

```diff
@@ -4412,6 +4412,18 @@ walkContinuationStackFrames(J9VMThread *currentThread, J9VMContinuation *continu
  */
 UDATA
 walkAllStackFrames(J9VMThread *currentThread, J9StackWalkState *walkState);
+
```

Comment:
minor: but I think all these functions should be implemented in the same file. ContinuationHelpers makes sense to me